src/226-Invert-Binary-Tree/226._Invert_Binary_Tree.py

```diff
@@ -0,0 +1,24 @@
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.left = None
+#         self.right = None
+
+
+class Solution:
+    def invertTree(self, root):
+        """
+        Invert a binary tree.
+
+        Args:
+            root(TreeNode): the root node of the given tree
+
+        Rerurns:
+            the root node of the Invert tree(TreeNode)
+        """
+        if root is None:
+            return None
+        
+        root.left, root.right = self.invertTree(root.right), self.invertTree(root.left)
+        return root
```

Comment:
You should add a new line at the end of this file

Reply:
OK，I'll do it @lingmagirl